components/camel-bindy/src/test/java/org/apache/camel/dataformat/bindy/model/date/BindyDatePatternCsvUnmarshallTest.java

```diff
@@ -167,7 +171,15 @@ public void setReceivedTime(LocalTime receivedTime) {
             this.receivedTime = receivedTime;
         }
 
-        public ReturnReason getReturnReason() {
+		public ZonedDateTime getDeletedDateTime() {
```

Comment:
Please run the build with the profile sourcecheck enable and fix the warnings you'll see